src/main/java/org/opentosca/container/client/model/PlanInstance.java

```diff
@@ -0,0 +1,32 @@
+package org.opentosca.container.client.model;
+
+import io.swagger.client.model.PlanInstanceDTO;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Wrapper class for PlanInstanceDTO
+ */
+@RequiredArgsConstructor
+public class PlanInstance {
+
+    private final PlanInstanceDTO plan;
+
+    public Long getServiceTemplateInstanceId() {
+        return plan.getServiceTemplateInstanceId();
+    }
+
+    public String getCorrelationId() {
+        return plan.getCorrelationId();
+    }
+
+    public PlanInstanceDTO.StateEnum getState() {
+        return plan.getState();
+    }
+
+    public PlanInstanceDTO.TypeEnum getType() {
```

Comment:
Not needed.

Reply:
Actually I did need this once in `ApplicationInstance.java` which is called in the `getManagementPlanInstances()` method.